### DISCUSSION TITLE
form-toggle with svg icon!?

### DISCUSSION THREAD

**Discussion by GregorWedlich** [saadeghi/daisyui#435] [Q&A] [answered]
Hi,

maybe I just missed it, but is there a way to place SVG icons inside a "form-toggle" switcher? I haven't found anything about it.

Example:

https://codepen.io/mkleung/pen/ymRowy

Best :)

**Comment by saadeghi** [comment, marked as answer]
There's a way: https://play.tailwindcss.com/h5i2wLNRbN  
Also follow [this issue](https://github.com/saadeghi/daisyui/issues/398) to get updates when a cleaner markup is added to daisyUI

**Reply by GregorWedlich** [reply]
Okay, I'll have a look at that!

But otherwise I am very excited about DaisyUI :)
